lib/publiccloud/utils.pm

```diff
@@ -146,6 +146,10 @@ sub register_openstack {
 # Validation for update repos
 sub validate_repo {
     my ($maintrepo) = @_;
+    if (is_sle_micro('>=6.0')) {
+        record_info("NEW MAINT MODE", "New style update, so can't validate repository");
```

Comment:
The message sounds a bit confusing for whom does not know about this. Maybe something like "Skip repo validation. Embargoed updates are not pushed in SL Micro 6+." or something similar.